ludwig/utils/strings_utils.py

```diff
@@ -287,6 +292,43 @@ def create_vocabulary(
     return vocab, str2idx, str2freq, max_line_length, pad_idx, padding_symbol, unknown_symbol
 
 
+def create_vocabulary_single_token(
```

Comment:
Instead of "single token", could we call this, more generically, `create_vocabulary_category()`?

It's possible to have category features that can be multiple tokens/words, i.e. `["cat", "dog", "not a cat", "striped cat"]`.